providers/aws/services/vpc/vpc_flow_logs_enabled/vpc_flow_logs_enabled.metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Provider": "aws",
+  "CheckID": "vpc_flow_logs_enabled",
+  "CheckTitle": "Ensure VPC Flow Logging is Enabled in all VPCs.",
+  "CheckType": ["Logging and Monitoring"],
+  "ServiceName": "vpc",
+  "SubServiceName": "flow_log",
+  "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
+  "Severity": "medium",
+  "ResourceType": "AwsEc2Vpc",
+  "Description": "Ensure VPC Flow Logging is Enabled in all VPCs.",
+  "Risk": "VPC Flow Logs provide visibility into network traffic that traverses the VPC and can be used to detect anomalous traffic or insight during security workflows.",
+  "RelatedUrl": "",
+  "Remediation": {
+    "Code": {
+      "CLI": "",
+      "NativeIaC": "",
+      "Other": "",
+      "Terraform": "https://docs.bridgecrew.io/docs/logging_9-enable-vpc-flow-logging#terraform"
```

Comment:
The same for the cloudmatos remediation

Reply:
It would be interesting including the solution using the web console which is available in the bridgecrew website.

Reply:
Done